src/main/java/HelloWorld.java

```diff
@@ -3,6 +3,7 @@
     public static void main(String[] args) {
 	String myTimeString = "test error";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "test error" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1192)